index.js

```diff
@@ -0,0 +1,87 @@
+const players = "http://localhost:3000/players"
+const playersDiv = document.querySelector(".player-container")
+const likeButtons = playersDiv.querySelectorAll('button')
+
+function playerBuilder(playerObj) {
+    const player = document.createElement('div')
+    player.id = `${playerObj.id}`
+    player.className = "player"
+    player.dataset.number = `${playerObj.number}`
+    player.innerHTML = `
+    <h3>
+    ${playerObj.name} (<em>${playerObj.nickname}</em>)
+    </h3>
+    <img src=${playerObj.photo} alt=${playerObj.name}>`
+
+    const likeButton = document.createElement('button')
+    likeButton.innerHTML = `${playerObj.likes} Likes!`
+    player.append(likeButton)
+    likeButton.addEventListener('click', incrementLikes)
+    playersDiv.append(player)
+}
+function allPlayers(arr) {
+    arr.forEach(playerBuilder)
+}
```

Comment:
I'd also suggest using another name for the parameter instead of `arr`, like `players` or `playerArray`, to describe the data.